src/pages/Setup.vue

```diff
@@ -536,6 +536,12 @@ export default {
   computed: {
     electrumConnected () {
       return this.$electrum.connected
+    },
+    isWalletValid () {
+      return ((this.seedData.type === 'new') || (this.seedData.type === 'import' && this.seedData.valid))
```

Comment:
Why do we need this?  Can we click continue right now with garbage data?

Reply:
Yeah, people were leaving the fields empty.